frontend/debian/cuttlefish-user.cuttlefish-operator.default

```diff
@@ -14,8 +14,7 @@ operator_https_port=1443
 #
 # The directory where the TLS certificate needed to establish the HTTPS
 # communication lives.
-# Defaults to "/etc/cuttlefish-common/operator/cert".
-# operator_tls_cert_dir=
```

Comment:
This change is unnecessary, that's indeed the default. Instead insert this line at the beginning of the `gen_cert()` function in cuttlefihs-operator.init:
```
operator_tls_cert_dir=${operator_tls_cert_dir:-/etc/cuttlefish-common/operator/cert}
```